editor/index.js

```diff
@@ -78,14 +70,16 @@ export function recreateEditorInstance( target, settings ) {
  * @param  {String}  id       Unique identifier for editor instance
  * @param  {Object}  post     API entity for post to edit
  * @param  {?Object} settings Editor settings object
+ * @param  {?Object} autosave Autosave post object
  * @return {Object}           Editor interface
  */
-export function createEditorInstance( id, post, settings ) {
+export function createEditorInstance( id, post, settings, autosave ) {
 	const target = document.getElementById( id );
 	const reboot = recreateEditorInstance.bind( null, target, settings );
+	setupHeartbeat();
```

Comment:
Would this be able to handle multiple editors on the same page? To that point, does our revised approach of a singleton editor store support this at all? (cc @youknowriad)

Reply:
Mainly heartbeat should only be initialized once, but yes - autosaving should work with multiple editors - its their listeners that 'trigger' the autosave by attaching and responding to data attached to the heartbeats - each editor would handle its own local data/state...

On merging to core, we may want to change how this initialized and consider adding the autosave customizations into autosave.js directly.

Reply:
Yeah, I guess multiple editor do not work, we should rename this function `initializeEditor` or something which suggested you can't call it twice for multiple instances. (not related to this PR though)